### DISCUSSION TITLE
打印 思路

### DISCUSSION THREAD

**Discussion by arit1988** [BYJRK/DotNet-Discussions#180] [WPF 开发]
一组数据，需要渲染到datagrid上，数据量比较大，涉及到多页。
这个需要打印，PrintVisual限制多。类似这种打印的，一般步骤哪些，具体什么思路呢

**Comment by BYJRK** [comment]
是否可以考虑单独将数据导出到一个 xlsx 表格中？

**Reply by arit1988** [reply]
暂时需求是可以预览和直接打印，暂不考虑导出到excel中再打印
